src/components/dialogs/AddBatchToQueueDialog.vue

```diff
@@ -0,0 +1,120 @@
+                <v-card-text>
+                    <v-text-field
+                        ref="inputFieldAddToQueueCount"
+                        v-model="input"
+                        :label="$t('Files.Count')"
+                        required
+                        hide-spin-buttons
+                        type="number"
+                        :rules="rules.count">
+                        <template #append-outer>
+                            <div class="_spin_button_group">
+                                <v-btn class="mt-n3" icon plain small @click="input++">
+                                    <v-icon>{{ mdiChevronUp }}</v-icon>
+                                </v-btn>
+                                <v-btn :disabled="input <= 1" class="mb-n3" icon plain small @click="input--">
+                                    <v-icon>{{ mdiChevronDown }}</v-icon>
+                                </v-btn>
+                            </div>
+                        </template>
+                    </v-text-field>
+                </v-card-text>
+                <v-card-actions>
+                    <v-spacer />
+                    <v-btn text @click="closeDialog">{{ $t('Files.Cancel') }}</v-btn>
+                    <v-btn color="primary" text type="submit" :disabled="!isValid">
+                        {{ $t('Files.AddToQueue') }}
+                    </v-btn>
+                </v-card-actions>
+            </v-form>
+        </panel>
+    </v-dialog>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component'
+import { Mixins, Prop, Watch } from 'vue-property-decorator'
+import BaseMixin from '@/components/mixins/base'
+import { mdiChevronDown, mdiChevronUp, mdiPlaylistPlus, mdiCloseThick } from '@mdi/js'
+
+@Component
+export default class AddBatchToQueueDialog extends Mixins(BaseMixin) {
+    mdiChevronDown = mdiChevronDown
+    mdiChevronUp = mdiChevronUp
+    mdiPlaylistPlus = mdiPlaylistPlus
+    mdiCloseThick = mdiCloseThick
+
+    /**
+     * Is the dialog currently visible?
+     */
+    @Prop({ type: Boolean, default: false }) declare readonly isVisible: boolean
+
+    /**
+     * Should there be a toast message after the file was added to the queue?
+     */
+    @Prop({ type: Boolean, default: false }) declare readonly showToast: boolean
+
+    /**
+     * Filename of the model to be added to the Queue.
+     */
+    @Prop({ type: String, required: true }) declare readonly filename: string
+
+    isValid = false
+    // because of the text field, the input is always a string
+    input: string = '1'
+
+    rules = {
+        count: [
+            (value: string) => !!value || this.$t('JobQueue.InvalidCountEmpty'),
+            (value: string) => parseInt(value, 10) > 0 || this.$t('JobQueue.InvalidCountGreaterZero'),
+        ],
+    }
+
+    async addBatchToQueueAction() {
+        const array = Array(parseInt(this.input)).fill(this.filename)
```

Comment:
@meteyou 

Note about the `parseInt` usage - in my unmodified code, I've added the `radix` argument on purpose, to force the "natural" parsing of inputs, as in "treat strings as natural number input". Typing something like "0x11" makes `parseInt` assume it's a hexadecimal, and while it is most likely a super rare case, I usually tend to prevent ambiguity in terms of parsing, that's why I've included it. Not sure if you know about this or not, and I also don't mind this being gone here (as I said, rather rare to see that kind of typing mistake made by the users), but just wanted to clarify in case you didn't know about this.

While this does not need any action, please note that the validation rule above still leverages the `radix = 10` argument, so the "wrong" (or rather, unnatural) input is impossible to be submitted. However, I think it's usually good to enforce same style of this function's usage, so if you agree with my take, I would recommend adding the radix back in some future refactoring task. This can also be enforced (autofixed even, I think) with ESLint rule: https://eslint.org/docs/latest/rules/radix (`always` option required, because `as-needed` incorrectly assumes that `10` is the default).

Reply:
ohh thx! i didn't know that! Do you want to add this eslint rule with a PR? I think this would be a good think to "fix" this issues in the complete project.

Reply:
Sure, I can pick this up, shouldn't be much of a problem :)